src/Tipsy.js

```diff
@@ -203,8 +221,13 @@ Tipsy.propTypes = {
    *
    * You may pass multiple triggers; separate them with a space. Pass an string with value "manual" to manually trigger the tooltip.
    */
-  trigger: PropTypes.string
+  trigger: PropTypes.string,
 
+  /**
+   * Customizable node for rendering the tipsy.
+   *
+   */
+  container: React.PropTypes.node,
```

Comment:
container should never be a Stateless Function. The Dom mounting tools doesn't recognize stateless functions as react component. In fact it maybe should be a React.PropTypes.element instead of a node.